src/inmanta/agent/config.py

```diff
@@ -102,54 +102,35 @@
     "config",
     "agent-deploy-interval",
     0,
-    "Either the number of seconds between two (incremental) deployment runs of the agent or a cron-like expression."
-    " If a cron-like expression is specified, a deploy will be run following a cron-like time-to-run specification,"
+    "Set the frequency and the granularity of deploy runs (i.e. only trigger a deploy for resources that have a "
+    " known divergence with their desired state)."
+    " When specified as an integer, this will set the number of seconds between two deploy runs on a per-resource basis."
+    " When specified as a cron-like expression, a global deploy (i.e. for all resources that have a known divergence with their"
+    " desired state) will be run following a cron-like time-to-run specification,"
     " interpreted in UTC. The expected format is ``[sec] min hour dom month dow [year]`` (If only 6 values are provided, they"
     " are interpreted as ``min hour dom month dow year``)."
-    " A deploy will be requested at the scheduled time. Note that if a cron"
-    " expression is used the :inmanta.config:option:`config.agent_deploy_splay_time` setting will be ignored."
+    " A deploy will be requested at the scheduled time."
     " Set this to 0 to disable the scheduled deploy runs.",
     is_time_or_cron,
     predecessor_option=agent_interval,
 )
-agent_deploy_splay_time = Option(
```

Comment:
Can we just remove this and `agent_repair_splay_time` here or should we first deprecate them ?

Reply:
I would remove them, the agent is gone anyways, this is now purely internal between server  and scheduler. 

We could even read it from the DB directly is we would like to.